### PR TITLE
Show beta license

### PR DESCRIPTION
**Pull request by mcalmer**
## What does this PR change?

Take the license text from the skelcd package and convert it to HTML before installing it.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: **internal**

- [x] **DONE**

## Test coverage
- No tests: **part of manual test**

- [x] **DONE**

## Links

- https://bugzilla.suse.com/show_bug.cgi?id=1164462

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"		 
- [ ] Re-run test "java_pgsql_tests"		 
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"		 
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"		 
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Comment by mcalmer** [comment]
@juliogonzalez what are we doing with this package in Uyuni?

**Comment by juliogonzalez** [comment]
> @juliogonzalez what are we doing with this package in Uyuni?

What do you mean, exactly? I don't think we can just remove it, right?

File: susemanager-branding-oss/susemanager-branding-oss.spec [hunk 1]
```diff
@@ -27,6 +27,7 @@ Source0:        %{name}-%{version}.tar.gz
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 BuildArch:      noarch
 Provides:       susemanager-branding = %{version}
+BuildRequires:  SUSE-Manager-Server-release
```
Comment (juliogonzalez) [inline susemanager-branding-oss/susemanager-branding-oss.spec]: Not sure I am understanding this. Do we want to avoid building the package on Uyuni?

**Comment by mcalmer** [comment]
@juliogonzalez SR https://build.opensuse.org/request/show/788221 removes susemanager-branding-oss from the pattern and I added a commit to this which exclude all archs when we build for openSUSE.

I think we can now merge this.

File: susemanager-branding-oss/susemanager-branding-oss.spec (hunk 1 above)
Comment (mcalmer) [reply susemanager-branding-oss/susemanager-branding-oss.spec]: This package contains the license files. For Uyuni we do not have license files. So we do not need this package.
Interestingly the ExcludeArch works with noarch packages.

File: susemanager-branding-oss/susemanager-branding-oss.spec [hunk 2]
```diff
@@ -26,6 +26,11 @@ Url:            https://github.com/uyuni-project/uyuni
 Source0:        %{name}-%{version}.tar.gz
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 BuildArch:      noarch
+%if 0%{?is_opensuse}
+ExcludeArch:    i586 x86_64 ppc64le s390x aarch64
```
Comment (juliogonzalez) [inline susemanager-branding-oss/susemanager-branding-oss.spec]: Looks good. I will see later if we can exclude submissions for this package.

**Pull request merged**